### PR TITLE
Article Flutterflow & Devops 

### PR DESCRIPTION
**Pull request by carter293**

### CODE REVIEW

File: _posts/2024-01-17-devops-and-flutterflow.markdown [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+layout: postv2
+font: serif
+title: "Role of FlutterFlow in Modern App Development: Insights for DevOps Teams"
+description: "FlutterFlow is a visual editor for building Flutter apps. It is a good starting point to move github and use DevOps practices to build out a full app"
```
Comment (Hamo69) [inline _posts/2024-01-17-devops-and-flutterflow.markdown:5]: what does "It is a good starting point to move github" mean?

File: _posts/2024-01-17-devops-and-flutterflow.markdown [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+- Good base for building out a POC or MVP
+- Can be used to build out a full app
+
+## What is FlutterFlow?
+
+- A visual editor for building Flutter apps
+- Extensive templates and pre-built components
+- Database integrations
+- Authentication
+- Export to code
```
Comment (Hamo69) [inline _posts/2024-01-17-devops-and-flutterflow.markdown:27]: what can you code in? Is it easily extensible?

File: _posts/2024-01-17-devops-and-flutterflow.markdown [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+## Drawbacks of FlutterFlow
+
+- DevOps practices really need github and CI/CD.
+- Heavy integration with Firebase and associated services
+
+## Conclusion
+
+- Great for MVP in product dev lifecycle
+- Production development needs github and CI/CD
```
Comment (Hamo69) [inline _posts/2024-01-17-devops-and-flutterflow.markdown:52]: still needs experts ;-)

File: _posts/2024-01-17-devops-and-flutterflow.markdown [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+- FlutterFlow is a good starting point but you can push the entire code base to github
+
+## DevOps and FlutterFlow
+
+- Automated Testing
+  - Flutter Integration tests and firebase test lab
+  - Need a pro plan
+- Version control
+  - Can ClickOps it but better to use git
```
Comment (Hamo69) [inline _posts/2024-01-17-devops-and-flutterflow.markdown:39]: why?

File: _posts/2024-01-17-devops-and-flutterflow.markdown [hunk 5]
```diff
@@ -0,0 +1,52 @@
+---
+layout: postv2
+font: serif
+title: "Role of FlutterFlow in Modern App Development: Insights for DevOps Teams"
+description: "FlutterFlow is a visual editor for building Flutter apps. It is a good starting point to move github and use DevOps practices to build out a full app"
+date: 2023-10-17
+highlight: monokai
+image: img/devopsflutterflow/cover.png
+author: Matthew Carter
+tags: [Trust policy, GitHub, GitHub Environments, Security, OIDC]
```
Comment (Hamo69) [inline _posts/2024-01-17-devops-and-flutterflow.markdown:10]: Product Dev?

File: _posts/2024-01-17-devops-and-flutterflow.markdown [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+## DevOps and FlutterFlow
+
+- Automated Testing
+  - Flutter Integration tests and firebase test lab
+  - Need a pro plan
+- Version control
+  - Can ClickOps it but better to use git
+- CI/CD & Trunk Based Development
+  - Not possible to enforces testing before publishing
```
Comment (Hamo69) [inline _posts/2024-01-17-devops-and-flutterflow.markdown:41]: interesting - seems like a big gap

**Review by Hamo69 — COMMENTED** [review]
Looks like a good outline

File: _posts/2024-01-17-devops-and-flutterflow.markdown (hunk 1 above)
Comment (carter293) [reply _posts/2024-01-17-devops-and-flutterflow.markdown:5]: it means I'm dyslexic

File: _posts/2024-01-17-devops-and-flutterflow.markdown (hunk 5 above)
Comment (carter293) [reply _posts/2024-01-17-devops-and-flutterflow.markdown:10]: Ill make a note to change these - ripped the template from another post...

File: _posts/2024-01-17-devops-and-flutterflow.markdown (hunk 6 above)
Comment (carter293) [reply _posts/2024-01-17-devops-and-flutterflow.markdown:79]: Might be possible but can't see it being possible yet - going to try the pro version free trail to test this and a couple other things.

File: _posts/2024-01-17-devops-and-flutterflow.markdown (hunk 2 above)
Comment (carter293) [reply _posts/2024-01-17-devops-and-flutterflow.markdown]: exports to flutter code, can't see any alternatives to that

File: _posts/2024-01-17-devops-and-flutterflow.markdown [hunk 7]
```diff
@@ -0,0 +1,166 @@
+---
+layout: postv2
+font: serif
+title: "Role of FlutterFlow in Modern App Development: Insights for DevOps Teams"
+description: "FlutterFlow is a visual editor for building Flutter apps. It is a good starting point for building applications, however, at a certain size applications may need to be moved GitHub to enable DevOps practises as well as robust, secure and consistent development"
+date: 2023-10-17
```
Comment (Hamo69) [inline _posts/2024-01-17-devops-and-flutterflow.markdown:6]: Can you please update to todays date?

**Review by Hamo69 — APPROVED** [review]
LGTM

**Pull request merged**